app-forensics/scalpel/scalpel-2.1_pre20210326.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="A high performance file carver"
+HOMEPAGE="https://github.com/sleuthkit/scalpel"
+GITHASH="35e1367ef2232c0f4883c92ec2839273c821dd39"
```

Referring to a git commit id as "git hash" is uncommon (at least in my perception). Even though it is technically a hash, I would suggest to use more familiar terms, e.g. `git-r3.eclass` uses `EGIT_COMMIT`. I would suggest to prefix it with `SCALPEL_`, to avoid name clashes, since it is a variable in global scope.

```suggestion
SCALPEL_COMMIT="35e1367ef2232c0f4883c92ec2839273c821dd39"
```

---

Okay, I can follow the argument with the variable in global scope. Will change that.